packages/cli/package.json

```diff
@@ -36,11 +36,15 @@
     "start": "pnpm configu"
   },
   "dependencies": {
-    "@configu/sdk": "workspace:*",
     "@configu/common": "workspace:*",
+    "@configu/sdk": "workspace:*",
+    "axios": "^1.7.4",
     "clipanion": "^4.0.0-rc.4",
     "consola": "^3.2.3",
+    "inquirer": "12.0.0",
```

Comment:
consider to use the next version:
https://www.npmjs.com/package/@inquirer/prompts

Reply:
Will open an issue for this, it's not necessary for this migration PR.

Reply:
Follow up issue: https://github.com/configu/configu/issues/579